Add Russian translation

ArkadiyVoronov opened the pull request:
<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #23098

ArkadiyVoronov committed:
Add Russian translation

greggubarev reviewed (changes requested):
@ArkadiyVoronov please change line 5 to more readable

ArkadiyVoronov committed:
fixed line 5

After review of @greggubarev @ArkadiyVoronov please change line 5 to more readable

greggubarev commented:
This PR was closed and then re-opened in an attempt to clear a specific Travis build failure.

greggubarev reviewed (approved):
LGTM 👍 :dart:

Thank you for the PR ✨ 🎉
 We're happy to accept these changes and look forward to your future contributions :clap:

Happy coding! :octocat:
